emails/utils.py

```diff
@@ -170,16 +170,16 @@ def sanitize_email(addr, encoding='ascii', parse=False):
     return addr
 
 
-def sanitize_address(addr, encoding='ascii'):
```

Comment:
from message.Message sanitize_address passing self.encoding argument, which defaults to 'utf-8'.